code/game/dna/genes/monkey.dm

```diff
@@ -46,9 +46,6 @@
 		animation.master = null
 		qdel(animation)
 
-	for (var/mob/living/simple_animal/borer/borer in Mo)
-		if (borer.controlling)
-			Mo.do_release_control(0)
```

Comment:
WHHHHYYYYYYYYYYYYYYYYYYY

Reply:
I added that block of code with the borer monkey PR specifically to prevent borers becoming human for free. Why did you just flat out remove it?